_data/social.yml

```diff
@@ -34,7 +27,7 @@ websites:
       img: bufferapp.png
       tfa: Yes
       sms: Yes
-      software: Yes
+     software: Yes
```

Comment:
Indentations are important. Please preserve them otherwise it's not a valid yaml file.